src/Cameras/XR/webXREnterExitUI.ts

```diff
@@ -117,7 +117,12 @@ export class WebXREnterExitUI implements IDisposable {
         });
     }
 
-    private constructor(private scene: Scene, options: WebXREnterExitUIOptions) {
+    /**
+     *
+     * @param scene babylon scene object to use
+     * @param options (read-only) version of the options passed to this UI
+     */
+    private constructor(private scene: Scene, public options: WebXREnterExitUIOptions) {

```

Comment:
correct one is:
/**
     * BLA BLA BLA
     * @param scene babylon scene object to use
     * @param options (read-only) version of the options passed to this UI
     */
    private constructor(private scene: Scene, 
      /** BBLA BLA BLA */
      public options: WebXREnterExitUIOptions) {